packages/core/src/Assets/SvgComponents/onboarding/cfds-mobile.svg

```diff
@@ -1 +1,15 @@
\ No newline at end of file
+<svg width="67" height="109" viewBox="0 0 67 109" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
None of the SVGs are minified. 
I suggest you to read through this file components/stories/icon/README.md

Reply:
My bad forgot to redo them after changing the svgs again